x-pack/plugins/triggers_actions_ui/public/application/components/json_editor_with_message_variables.tsx

```diff
@@ -31,7 +31,8 @@ interface Props {
 }
 
 const { useXJsonMode } = XJson;
-const xJsonMode = new XJsonMode();
+// Source used to insert text imperatively into the code editor
+const EDITOR_SOURCE = 'json-editor-with-message-variables';
```

Comment:
Sorry, this is probably down to me not being familiar with monaco, but what is the impact of this source string?
Does it denote some kind of unique identifier?
What happens if there are multiple monaco editors on the page and they all use this same string?

Reply:
gotcha, I think an improved comment to this effect would help here

Reply:
>Multiple editors do no share the same editor reference, so you can use the same source ID

Cool :) 
Lets verify ;) but sound good 👍

Reply:
I've tried several on the same page and it looks great :)